app/js/lib/foxbox.js

```diff
@@ -63,6 +64,7 @@ export default class Foxbox extends Service {
     super();
     this.settings = new FoxboxSettings();
     this.db = new FoxboxDb();
+    this.qr = new FoxboxQr();
```

Comment:
I'm really not sure about adding another dependency to the foxbox.js lib.
My plan for the future is to make this lib an independent project, whose goal is to ease communication with the box by abstracting away authentication and API calls.
What you're achieving here shouldn't be taken care of by this script IMO.